scripts/build.py

```diff
@@ -1026,17 +1059,23 @@ def _verify_build(input_dirnames, output_dirnames, file_hashes):
         hashes are inserted.
 
         Args:
-            input_dirnames: list(str). List of directory paths that contain
-                source files.
+            input_dirnames: list(str|list(str)). List of directory paths that
```

Comment:
No, don't introduce type polymorphism here, for the same reasons as before.

Reply:
Updated.